Clappr.xcodeproj/project.pbxproj

```diff
@@ -731,13 +780,21 @@
 			targets = (
 				607FACCF1AFB9204008FA782 /* Clappr_Example */,
 				607FACE41AFB9204008FA782 /* Clappr_Tests */,
-				96D3627B1D41337600CCB866 /* Clappr */,
+				96D3627B1D41337600CCB866 /* Clappr-iOS */,
```

Comment:
@igoriols not sure if we really have to rename Clappr to Clappr-iOS target, but in case you proceed with this I recommend to also rename: Clappr-iOS-Example, Clappr-iOS-Tests, Clappr-iOS-UITests. To simplify this I think you can avoid renaming Clappr now

Reply:
@jchahoud Ok, I'm going to revert this change.